### PR TITLE
Disable debug mode when using --json output.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: anaconda_anon_usage/utils.py [hunk 1]
```diff
@@ -6,7 +6,9 @@
 from threading import RLock
 
 DPREFIX = os.environ.get("ANACONDA_ANON_USAGE_DEBUG_PREFIX") or ""
-DEBUG = bool(os.environ.get("ANACONDA_ANON_USAGE_DEBUG")) or DPREFIX
+DEBUG = (
```
Comment (jezdez) [inline anaconda_anon_usage/utils.py:9]: @mcg1969 is this what you had in mind?

Comment (mcg1969) [reply anaconda_anon_usage/utils.py:9]: Not quite, we should be using `context.json`, IMO
I didn't appreciate you were working on a PR too. I've got one coming that has some tests baked in too. But I think we need to discuss how it should be have

Comment (jezdez) [reply anaconda_anon_usage/utils.py:9]: Sorry about that, I should have mentioned it! Won't happen again

**Comment by jezdez** [comment]
Closing in favor of #36

**Pull request closed without merging**